_security-analytics/sec-analytics-config/log-types.md

```diff
@@ -1,51 +1,79 @@
 ---
 layout: default
-title: Supported log types
+title: Working with log types
 parent: Setting up Security Analytics
-nav_order: 16
+nav_order: 14
+redirect_from: 
+   - /security-analytics/sec-analytics-config/custom-log-type/
 ---
 
+# Working with log types
 
-# Supported log types
+Log types represent the different sources of data used for threat detection in Security Analytics. Log types are useful for categorizing or prepopulating [Detection rules]({{site.url}}{{site.baseurl}}/security-analytics/sec-analytics-config/detectors-config/) when creating detectors from your source.
 
-Logs contain raw data about events that happen throughout a system and in its separate parts. As of OpenSearch 2.11, log types are grouped by category to help select, filter, and search the log types. 
+Log types come from the following sources:
 
-To navigate to the **Log types** page, select **Log types** under **Detectors** in the **Security Analytics** navigation menu. The page shows the name of the log type, its description, its category, and identifies whether it's a standard OpenSearch-defined log type or a custom log type. The following image shows the **Log types** landing page with the Category column selected and the **Category** filter you can use to filter the list by the category. 
+- [Standard log types](#standard-log-types): Security Analytics automatically generates a list of data sources, along with their field mappings and rules, based on the data indexed from each source.
+- [Custom log types](#creating-custom-log-types): When your data falls outside the [standard log types]({{site.url}}{{site.baseurl}}/security-analytics/sec-analytics-config/log-types/), you can create a user-defined log type. For enhanced threat detection, Security Analytics supports the integration of custom log types.
```

Comment:
"Falls outside" is a phrasal verb, so let's replace it with something more precise. Maybe something like "When your data cannot be categorized as one of the standard log types"?